src/DynamoCoreWpf/ViewModels/Menu/TrustedPathViewModel.cs

```diff
@@ -171,6 +171,7 @@ private void CommitChanges(object param)
         internal void InitializeTrustedLocations()
         {
             TrustedLocations = new ObservableCollection<string>(settings?.TrustedLocations ?? new List<string>());
+            RaisePropertyChanged("");
```

Comment:
what does this do?

Reply:
RaisePropertyChanged with no specific property notifies the UI all the properties data had changed so the user can see the new settings just after importing them